package.json

```diff
@@ -58,30 +57,32 @@
     "@types/jasmine": "~3.3.8",
     "@types/jasminewd2": "~2.0.6",
     "@types/node": "^12.20.55",
-    "@typescript-eslint/eslint-plugin": "5.3.0",
-    "@typescript-eslint/parser": "5.3.0",
+    "@typescript-eslint/eslint-plugin": "5.29.0",
+    "@typescript-eslint/parser": "5.29.0",
     "copyfiles": "^2.4.1",
     "dotenv": "^10.0.0",
-    "eslint": "^8.2.0",
+    "eslint": "^8.18.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jsdoc": "^39.3.3",
+    "eslint-plugin-prefer-arrow": "^1.2.3",
     "http-server": "^0.12.3",
     "husky": "^8.0.1",
-    "jasmine-core": "~3.5.0",
+    "jasmine-core": "^3.99.1",
     "jasmine-spec-reporter": "~5.0.0",
     "karma": "~6.3.16",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coverage-istanbul-reporter": "~3.0.2",
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "^1.5.0",
     "karma-teamcity-reporter": "^1.1.0",
-    "ng-packagr": "^13.2.1",
-    "protractor": "~7.0.0",
+    "ng-packagr": "^14.0.2",
+    "semantic-release": "^19.0.3",
     "semantic-release-cli": "^5.4.4",
     "ts-node": "~8.0.2",
-    "typescript": "~4.5.5",
-    "semantic-release": "^19.0.3"
+    "typescript": "~4.7.4"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/BugSplat-Git/bugsplat-ng.git"
   }
-}
+}
```

Comment:
seeing a few of these no new lines. not sure if that is important to you

Reply:
tried fixing this but since i couldn't get it within a few minutes i decided to bail!